stateMachine/stateMachineHelpers.js

```diff
@@ -0,0 +1,64 @@
+const { helpers } = require('brave-alert-lib')
+const redis = require('../db/redis')
+const RADAR_TYPE = require('../RadarTypeEnum')
+
+async function movementAverageOverThreshold(radarType, locationid, movementThreshold) {
+  if (radarType === RADAR_TYPE.XETHRU) {
+    try {
+      const xeThruhistory = await redis.getXethruWindow(locationid, '+', '-', 15) // Array of last 15 readings from this location
+      if (xeThruhistory.length === 0) {
+        return false
```

Comment:
Maybe add an error log when this happens because it shouldn't, right? Or will it, the first few heartbeats after the sensor is added?